src/Sulu/Component/Rest/ListBuilder/Doctrine/FieldDescriptor/DoctrineGroupConcatFieldDescriptor.php

```diff
@@ -61,9 +71,15 @@ public function getSelect()
     }
 
     /**
-     * Returns all the joins required for this field.
-     *
-     * @return DoctrineJoinDescriptor[]
+     * {@inheritdoc}
+     */
+    public function getSimpleSelect()
```

Comment:
Would probably be better, but that's not what I am talking about.

If you use that for the where clause, and you use the field name in here, it will probably crash if the query does not include the select call, because it does not know about this name.